source/extensions/tracers/lightstep/lightstep_tracer_impl.cc

```diff
@@ -36,6 +37,14 @@ void LightStepLogger::operator()(lightstep::LogLevel level,
 LightStepDriver::LightStepTransporter::LightStepTransporter(LightStepDriver& driver)
     : driver_(driver) {}
 
+// If the default min_flush_spans value is too small, spans can be dropped between reports.
+// Hence, we choose a number that's large enough, but divide by the number of CPUs since Envoy
+// creates separate tracers for each thread.
+//
+// See https://github.com/lightstep/lightstep-tracer-cpp/issues/106
+const size_t LightStepDriver::default_min_flush_spans =
```

Comment:
Should the concurrency value passed in through the command line be used first then default to the hardware concurrency if not specified?
https://github.com/envoyproxy/envoy/blob/c92a3017017dfa31241dec13dc6a1479090318d0/source/server/options_impl.cc#L53

Reply:
@rnburn I gave this more thought and now believe we should just fill in a constant here. Envoy's architecture is that every CPU is an independent worker, so as long as the amount of memory available scales with the number of processors, there's no reason to limit the buffering of LightStep spans according to processor count.

Will you adjust this to use a fixed constant of 200? That avoids the question Constance asks here.

Reply:
Update to use 200